### PR TITLE
More v3 docs

### PR DESCRIPTION
**Pull request by SethDavenport**
- Changelog details
- Better DI action-creator example showing how `redux-thunk` is redundant in Angular 2.
- Demonstrate `redux-logger` and `redux-localstorage` as community-provided middleware and enhancer in example app.

Got rid of redux-thunk.

### CODE REVIEW

**Comment by SethDavenport** [comment]
@e-schultz

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,61 +1,166 @@
...
- ```ts
- import { NgRedux } from 'ng2-redux';
- // ... 
- export class MyComponent {
-   constructor(private ngRedux: NgRedux) {
-   }
- }
- ```
+
+```typescriptts
```
Comment (igor-ka) [inline CHANGELOG.md]: typo

File: README.md [hunk 2]
```diff
@@ -362,86 +362,97 @@ export class Counter {
...
+  decrement(): void {
+    this.ngRedux.dispatch({ type: CounterActions.DECREMENT_COUNTER });
   }
 
-  // Just a regular, synchronous action creator.
-  logoutUser() {
-    return { type: LOGOUT_USER };
+  // Async action.
+  incrementAsync(delay: number = 1000): void {
+    setTimeout(this.increment.bind(this), delay);
```
Comment (igor-ka) [inline README.md]: Not sure if there are drawbacks to using bind, but alternatively we could define increment as an arrow function on the class. This way we can preserve the context of `this`.

I am sure you guys have thought about this. I am just curious why `bind` is a better option?

**Comment by igor-ka** [comment]
@SethDavenport @e-schultz this is getting better and better. There is a small typo, but otherwise looks great. :shipit:

File: CHANGELOG.md (hunk 1 above)
Comment (SethDavenport) [inline CHANGELOG.md]: oops

**Pull request merged**